scripts/utils/sort-object/sort-object.constants.js

```diff
@@ -0,0 +1,8 @@
+/**
+ * Special character used to prevent JavaScript auto-sorting in non-serialized JSON Objects.
+ */
+const SORT_LOCK_CHARACTER = "�";
```

Comment:
not quite understanding the usage of this - if that is amended, u stop sorting everything underneath? is that correct?

Reply:
JavaScript automatically sorts object keys.

If a JavaScript object is sorted via a reducer to be:

```js
const obj = {
  '00': 'value',
  '05': 'value',
  '50': 'value',
  '100': 'value',
};
```

When stringifying or just printing the line to console, it appears as so:

```js
console.log(JSON.stringify(obj, null, 2));

// output
{
  "50": "value",
  "100": "value",
  "00": "value",
  "05": "value"
}
```

For some reason, the keys that start with `0` get pushed to the end. Feel free to test this locally 😉